snapshot/git/gitdb/schemes.go

```diff
@@ -0,0 +1,65 @@
+package gitdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// GitDB uses several different Schemes to identify, upload and download Values.
+// schemeValue abstracts these concerns.
+type schemeValue interface {
```

Comment:
I was thinking this refers to one value in a scheme. So the localValue object refers to one value specifically. I'd expect localScheme to be for the whole scheme.